packages/dashboard-frontend/src/services/workspace-client/devworkspace/__tests__/devWorkspaceClient.changeWorkspaceStatus.spec.ts

```diff
@@ -35,6 +35,7 @@ describe('DevWorkspace client, changeWorkspaceStatus', () => {
 
   afterEach(() => {
     jest.resetAllMocks();
+    jest.restoreAllMocks();
```

Comment:
`restoreAllMocks` should be enough because it does what `resetAllMocks` does, and additionally restores non-mocked implementation.

```suggestion
    jest.restoreAllMocks();
```